StartEmailValidationProcess/handler.ts

```diff
@@ -99,13 +107,32 @@ export function StartEmailValidationProcessHandler(
     );
 
     const dfClient = df.getClient(context);
-    await dfClient.startNew(
-      "EmailValidationProcessOrchestrator",
-      undefined,
-      emailValidationProcessOrchestartorInput
-    );
-
-    return ResponseSuccessAccepted();
+    return taskEither
+      .of(makeStartEmailValidationProcessOrchestratorId(fiscalCode, email))
+      .chain(orchId =>
+        isOrchestratorRunning(dfClient, orchId).foldTaskEither<
+          Error,
+          IResponseSuccessAccepted
+        >(fromLeft, _ =>
+          _.isRunning
+            ? taskEither.of(void 0)
+            : tryCatch(
+                () =>
+                  dfClient.startNew(
+                    "EmailValidationProcessOrchestrator",
+                    orchId,
+                    emailValidationProcessOrchestartorInput
+                  ),
+                toError
+              )
+        )
+      )
+      .mapLeft(err => {
+        context.log.error(`${logPrefix}|ERROR=${String(err)}`);
+        throw new Error(String(err));
```

Comment:
Not a big fan of this `throw`... We should return a response error here

Reply:
Yep... but API specs does not provide 500 at the moment. For instance, if we add 500 to API specs we must modify also io-backend and io-app.

Reply:
we must add 500 to fn-app API specs, the backend seems to alreaady handle it here:
https://github.com/pagopa/io-backend/blob/master/api_backend.yaml#L345